.github/CONTRIBUTING.md

```diff
@@ -103,68 +101,66 @@ Additional references:
-
 * When you create a pull request,
   including a summary of what's included in your changes and
   if the changes are related to an existing GitHub issue,
   please reference the issue in pull request description (e.g. ```Closes #11```).
   See [this][closing-via-message] for more details.
-
 * If the change warrants a note in the [changelog](../CHANGELOG.MD)
   either update the changelog in your pull request or
   add a comment in the PR description saying that the change may warrant a note in the changelog.
   New changes always go into the **Unreleased** section.
   Keeping the changelog up-to-date simplifies the release process for Maintainers.
-  An example:
-    ```
-    Unreleased
-    ----------
-
-    * `Update-Item` now supports `-FriendlyName`.
-    ```
-    Please use the present tense and imperative mood when describing your changes:
-
-      * Instead of "Adding support for Windows Server 2012 R2", write "Add support for Windows Server 2012 R2".
-
-      * Instead of "Fixed for server connection issue", write "Fix server connection issue".
-
-    This form is akin to giving commands to the code base,
-    and is recommended by the Git SCM developers.
-    It is also used in the [Git commit messages](#common-engineering-practices).
-
-    Also, if change is related to a specific resource, please prefix the description with the resource name:
-
-      * Instead of "New,parameter 'ConnectionCredential' in New-SqlConnection",
-        write "New-SqlConnection: added parameter 'ConnectionCredential'".
+  An example (with an associated PR #):
+
+  ```markdown
+  Unreleased
+  ----------
+
+  * `Update-Item` now supports `-FriendlyName` (#1234).
+  ```
+
+* Please use the present tense and imperative mood when describing your changes:
+  * Instead of "Adding support for Windows Server 2012 R2", write "Add support for Windows Server 2012 R2".
+  * Instead of "Fixed for server connection issue", write "Fix server connection issue".
+
+  This form is akin to giving commands to the code base
+  and is recommended by the Git SCM developers.
+  It is also used in the [Git commit messages](#common-engineering-practices).
+* If the change is related to a specific resource, please prefix the description with the resource name:
+  * Instead of "New parameter 'ConnectionCredential' in New-SqlConnection",
+  write "New-SqlConnection: added parameter 'ConnectionCredential'".
+* If your change warrants an update to user-facing documentation,
+  a Maintainer will add the `Documentation Needed` label to your PR and add an issue to the [PowerShell-Docs repository][PowerShell-Docs],
+  so that we make sure to update official documentation to reflect your contribution.
+  As an example, this requirement includes any changes to cmdlets (including cmdlet parameters) and features which have associated about_* topics.
+  While not required, we appreciate any contributors who add this label and the issue themselves.
```

Comment:
This sentence reads oddly.  This is minor, but suggestion ` While not required, we appreciate any contributors who add this label and create the issue themselves.`